dbaccess/transaction.go

```diff
@@ -182,6 +182,39 @@ func AcceptedTransactionsByBlockID(ctx database.Context, blockID uint64, preload
 	return transactions, nil
 }
 
+// TransactionDoublespends retrieves transactions, that have at least one input which is the same as in transaction that has the provided hash
+func TransactionDoublespends(ctx database.Context, transactionHash string, preloadedFields ...dbmodels.FieldName) ([]*dbmodels.Transaction, error) {
+	db, err := ctx.DB()
+	if err != nil {
+		return nil, err
+	}
+
+	origInputs := db.Model(&dbmodels.TransactionInput{}).
```

Comment:
Please refrain from abbreviations where possible (both in variable here name and in query parameter orig_inputs).